src/lineprinter.rs

```diff
@@ -1596,15 +1596,10 @@ mod tests {
             ..default_line_printer(&mut term)
         };
 
-        for (available_space, used_space, expected) in vec![
-            (14, 11, r#"[ 3 items ]"#),
-            (4,  3, r#"[…]"#),
-            (2,  0, r#""#),
-        ]
-        .into_iter()
+        for (available_space, used_space, expected) in
+            vec![(14, 11, r#"[ 3 items ]"#), (4, 3, r#"[…]"#), (2, 0, r#""#)].into_iter()
```

Comment:
I think the linter is being overzealous here, and in the rest of the `test_` functions in this commit. It's way harder to read this `vec![...]` on a single line. They were on their own lines with specific spacing to make it easier for humans to read.